assets/js/src/newsletters/listings/standard.jsx

```diff
@@ -5,7 +5,7 @@ import MailPoet from 'mailpoet';
 import Hooks from 'wp-js-hooks';
 import PropTypes from 'prop-types';
 
-import confirmAlert from 'common/confirm_alert.jsx';
+import confirmAlertModal from 'common/confirm_alert_modal.jsx';
```

Comment:
We should use the `confirm_alert.jsx` here. That was the whole reason to split it up so that we can use it inside other React components.